### ISSUE TITLE
[yesh + infinitive] and [al+PRON + infinitive]

### ISSUE THREAD

**Issue by shirawigi** [IAHLT/UD_Hebrew#10]
@amir-zeldes
What is the correct way to tag the structures [yesh + infinitive] and [al+PRON + infinitive]?
example for the first structure from the data:
1. **יש להפריד** בין ניהול ה בנקים לבין ניהול קרנות ה נאמנות ו קופות ה גמל

examples for the second structure from the data:
1. **היה עליך להלך** ביניהם בזהירות רבה
2. **עלינו להציל** אותם

Thanks,
Shira

**Comment by amir-zeldes** [comment]
I think it would be most consistent to treat it like efshar and tsarix, so it would be:

csubj(yesh, lehafrid)

csubj(nu, lehatsil)
case(nu, aley)

**Comment by NathanD38** [comment]
@amir-zeldes 
Previously, in a comment thread on Google Docs Guidelines, you recommended treating 
yesh+infinitive as AUX+VERB, the infinitive being the root.
This analysis is analoguous to the one of "tsarix"+infinitive, which, at least semantically, is similar.

**Comment by amir-zeldes** [comment]
I'm not sure about that thread, but for the impersonal case both tsarix and yesh should be the root IMO, since the predication is about an infinitive clause X (a csubj), stating that it is necessary. Compare English annotations:

  * necessary: http://match.grew.fr/?corpus=UD_English-EWT@dev&custom=614a15c62e612
  * possible: http://match.grew.fr/?corpus=UD_English-EWT@dev&custom=614a15c62e612

Or Russian можно + INF:

http://match.grew.fr/?corpus=UD_Russian-SynTagRus@2.8&custom=614a165484983&eud=yes

I do think the AUX analysis is right for things like "ani tsarix lalexet" (where subject is "ani" and the predication is an expanded modalized form of "go"; English etc. also behave this way in UD due to lexicocentrism)

**Comment by shirawigi** [comment]
@amir-zeldes

regarding [al+PRON + infinitive], what should be the root? 
In this sentence: **עלינו להציל אותם** , the root was the PRON **nu**, but that seems odd to me.

**Comment by amir-zeldes** [comment]
No, that sounds right (predication: "lehatsil otam" is "aleinu"). Consider that it could be a noun too (al ha-memshala...)
